themes/default/gulpfile.js

```diff
@@ -212,7 +227,7 @@ gulp.task('make-js', ['make-js-components', 'make-js-npm'], function() {
 		.pipe(sourcemaps.init())
 		.pipe(concat('script.js'))
 		.pipe(gulp.dest('js/src/'))
-		.pipe(uglify())
+		.pipe(gulpif(process.env.NODE_ENV !== 'dev', uglify()))
```

Comment:
Right, the bit I wasn't getting was where we were setting ENV variables. The answer is: its an optional value that  we set in the makefile (if you happen to be using it.). :+1: